.github/workflows/helm-test-workflow.yml

```diff
@@ -69,6 +70,7 @@ jobs:
       DEPLOYMENT_TYPE: "helm"
       HELM_REPO_PATH: "../../../../helm-chart"
       INSTALL_OPERATOR: "true"
+      TEST_VPC_ENDPOINT_URL: ${{ secrets.TEST_VPC_ENDPOINT_URL }}
```

Comment:
Can we instead use env variable as part of `.env` file in our project? Just don't want to fall into the trap again where we have to update the secrets again and we dont know why the tests failed when someone changed the secrets or something like that.

Reply:
@vivekr-splunk and I discussed it. I proposed using this method in order to not have a VPC endpoint on an open sourced repo. In the future we can store these in a separate repo(gitlab?).

Reply:
If the env method is preferred I can switch to that as well :)